input/pages/changes.md

```diff
@@ -3,8 +3,13 @@
 {: #changes}
 
 This page details changes made in each version of the Using CQL with FHIR Implementation Guide.
+### STU1 Reconciliation Changes (version 1.0.0-ballot)
 
-### Initial STU Ballot (version 1.0.0-ballot)
+Change Summary
+This ballot made the following major changes:
```

Comment:
Move this change description to the ballot change section and add language describing the "reconciliation changes"

Reply:
Done